main.go

```diff
@@ -433,6 +433,14 @@ func main() {
 					Name:  "symbols, s",
 					Usage: "Use symbols in the password",
 				},
+				cli.BoolFlag{
+					Name:  "xkcd, x",
+					Usage: "Use multiple random english words as password, separated by space",
+				},
+				cli.BoolFlag{
+					Name:  "xkcdo, xo",
+					Usage: "Use multiple random english words as password, no separator but CamelCased",
```

Comment:
While it is certainly debatable, this is not what I recognize as `camelCase` (lower first letter), but rather `ClassCase` or `PascalCase`: https://en.wikipedia.org/wiki/Camel_case#Variations_and_synonyms

Perhaps not an important comment given the discussion of adding a separator flag. 

Additionally, you should capitalize English.

Reply:
I think the name Camel Case is OK. So far I never met someone making a difference between `camelCase` and other variations like `ClassCase`.

I agree that we should credit the original XKCD comic. Probably somewhere in the Readme.

Reply:
Yeah, fair enough. If we add a separator flag instead, this comment is likely to go away anyhow, so any confusion about what the case means will be a non-issue.